src/sentry/models/integrations/integration.py

```diff
@@ -134,3 +136,13 @@ def add_organization(self, organization: RpcOrganization, user=None, default_aut
                 },
             )
             return False
+
+    def uninstall(self):
```

Comment:
We might wanna rename this and other references to `disable` instead of `uninstall` since it's not actually removing the integration, right?